project/src/main/java/com/google/sps/utilities/SpannerTestTasks.java

```diff
@@ -36,7 +36,11 @@ public static void setup() throws InterruptedException, ExecutionException {
                 + "  NumSpotsLeft               INT64 NOT NULL,"
                 + "  RequiredSkills             ARRAY<STRING(MAX)>,"
                 + "  Volunteers                 ARRAY<STRING(MAX)>"
-                + ") PRIMARY KEY (VolunteeringOpportunityID)");
+                + ") PRIMARY KEY (VolunteeringOpportunityID)",
```

Comment:
nit: Format with tabs so we can quickly see the columns and types

Reply:
Done.